code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -2079,6 +2079,22 @@
 	if(volume >= 3)
 		if(!(locate(/obj/effect/decal/cleanable/greenglow) in T))
 			new /obj/effect/decal/cleanable/greenglow(T)
+			
+/datum/reagent/diamond
+	name = "Diamond dust"
+	id = DIAMOND
```

Comment:
please use DIAMONDDUST for clarity (hence your changes in gambling code)

Reply:
I used DIAMOND to have consistency with the rest of direct material code, since Gold isn't GOLDDUST but rather GOLD, same with silver, plasma, uranium and phazon.

Reply:
I think your intentions are good but less ambiguous defines are better

Reply:
Fair enough.

Reply:
That should do it.